src/app/checkout/step-2/step-2.component.js

```diff
@@ -117,6 +117,9 @@ class Step2Controller {
   }
 
   getContinueDisabled () {
```

Comment:
I know you didn't name it, but should we change this function name to `isContinueDisabled()`?